### PR TITLE
PA-1678 Land Classification form updates, includes new usage form.

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: frontend/src/components/common/Label.tsx [hunk 1]
```diff
@@ -1,16 +1,19 @@
 import * as React from 'react';
 import './Label.scss';
 interface ILabelProps {
-  children?: string;
+  content?: string;
```
Comment (devinleighsmith) [inline frontend/src/components/common/Label.tsx:4]: using children as a string is confusing given the react naming convention for nested child components.

File: frontend/src/components/common/Label.tsx [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
   required?: boolean;
 }
 
-/** generic inline label element */
-export const Label = (props: ILabelProps | null | undefined) => {
+/** Generic inline label element */
+export const Label: React.FunctionComponent<ILabelProps & React.HTMLAttributes<HTMLDivElement>> = ({
```
Comment (devinleighsmith) [inline frontend/src/components/common/Label.tsx:9]: Labels should support all of the standard html attribute tags.

File: frontend/src/components/maps/__snapshots__/BuildingPopupView.test.tsx.snap [hunk 3]
```diff
@@ -42,7 +42,7 @@ exports[`building popup view renders correctly 1`] = `
           onClick={[Function]}
         >
           <p
-            className="label"
+            className="label undefined"
```
Comment (devinleighsmith) [inline frontend/src/components/maps/__snapshots__/BuildingPopupView.test.tsx.snap:45]: You can safely ignore this as it has been corrected in a future revision.

File: frontend/src/components/common/Label.tsx (hunk 2 above)
Comment (jdtoombs) [reply frontend/src/components/common/Label.tsx:10]: Ooh good call

**Review by jdtoombs — APPROVED** [review]

**Pull request closed without merging**